anabeesh/src/main/java/com/cs18/anabeesh/api/UserRegistrationQueries.java

```diff
@@ -0,0 +1,18 @@
+package com.cs18.anabeesh.api;
+
+import com.cs18.anabeesh.store.model.register.RegisterUserRequest;
+import com.cs18.anabeesh.store.model.register.RegisterUserResponse;
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
+
+/**
+ This interface holds all the queries needed to register with Anabeesh's API
+ */
+
+public interface UserRegistrationQueries {
```

Comment:
Mate3melsh class l kol 7aga lwa7daha, e3mel msln class esmaha `APIService` feeha kol da

Reply:
Class kbeer l retrofit we gowah 2 inner interface ?

Reply:
Ana asdi interface ya3ni m3lish msh class 7arfeyan

Reply:
done.3mlto el class we gowah el 2 interfaces